lib/sidekiq.rb

```diff
@@ -137,6 +137,10 @@ def self.configure_client
     yield default_configuration unless server?
   end
 
+  def self.serialize_as(type)
```

Comment:
Not a fan of the global method. Putting it in S::Config should allow us to do:

```ruby
Sidekiq.configure_client do |cfg|
  cfg.serialize_as(...)
end
```

Better?

Reply:
We will have to be careful to call `serialize_as` on both the client and server though. A retry might blow up otherwise.

Reply:
I pulled it out.